irepela/homework_3/conversion.py

```diff
@@ -0,0 +1,50 @@
+def convert_binary_to_decimal(binary_str: str) -> int:
+    """
+      Converts binary string to decimal number and returns result.
+
+     Args:
+         binary_str (str): Binary string.
+
+     Returns:
+         int: Decimal number.
+     """
```

Comment:
in example you could provide at least 1 - 2 specific examples  with actual values

```
def convert_binary_to_decimal(binary_str: str) -> int:
    """
      Converts binary string to decimal number and returns result.
      
      Usage Example:
           convert_binary_to_decimal('0b1010101') == ...

     Args:
         binary_str (str): Binary string.
     Returns:
         int: Decimal number.
     """

```